keras_cv/models/resnet_v1_test.py

```diff
@@ -51,8 +52,54 @@ def test_application_variable_input_channels(self, app, last_dim, args):
         super()._test_application_variable_input_channels(app, last_dim, args)
 
     @parameterized.parameters(*MODEL_LIST)
-    def test_model_can_be_used_as_backbone(self, app, last_dim, args):
-        super()._test_model_can_be_used_as_backbone(app, last_dim, args)
+    def test_model_serialization_tf(self, app, last_dim, args):
+        super()._test_model_serialization(
+            app, last_dim, args, save_format="tf", filename="model"
+        )
+
+    @parameterized.parameters(*MODEL_LIST)
+    def test_model_serialization_keras_format(self, app, last_dim, args):
+        if version.parse(tf.__version__) >= version.parse("2.12.0-dev0"):
+            super()._test_model_serialization(
+                app,
+                last_dim,
+                args,
+                save_format="keras_v3",
+                filename="model.keras",
+            )
+
+    def test_create_backbone_model_from_application_model(self):
+        model = resnet_v1.ResNet50(
+            include_rescaling=False,
+            include_top=False,
+            classes=2048,
```

Comment:
`classes` here shouldn't do anything, since `include_top` is False.

I would customarily suggest that we add an error if `classes` is specified when `include_top=False`, but I think we will soon slightly restructure how classification models are made from these applications-style models, so for now it's fine as-is.